nautobot/extras/templates/extras/tag.html

```diff
@@ -31,6 +31,14 @@
                             <span class="label color-block" style="background-color: #{{ object.color }}">&nbsp;</span>
                         </td>
                     </tr>
+                    <tr>
+                        <td>Content Type(s)</td>
+                        <td>
+                            {% for ct in content_types %}
+                                {{ ct }}<br>
+                            {% endfor %}
```

Comment:
I recognize this is following a pattern established in `status.html` but I think it would be better done as follows:

```suggestion
                            <ul class="list-unstyled">
                                {% for ct in content_types %}
                                    <li>{{ ct }}</li>
                                {% endfor %}
                            </ul>
```